connector/profiler/README.md

```diff
@@ -60,36 +68,28 @@ Then enable the profiling in the configuration.
   <td>4.0.0</td>
 </tr>
 <tr>
-  <td><code>spark.executor.profiling.localDir</code></td>
+  <td><code>spark.profiler.localDir</code></td>
   <td><code>.</code> i.e. the executor's working dir</td>
   <td>
    The local directory in the executor container to write the jfr files to. If not specified the file will be written to the executor's working directory. Users should ensure there is sufficient disk space available on the system as running out of space may result in corrupt jfr file and even cause jobs to fail on systems like K8s.  
   </td>
   <td>4.0.0</td>
 </tr>
 <tr>
-  <td><code>spark.executor.profiling.options</code></td>
+  <td><code>spark.profiler.asyncProfiler.args</code></td>
```

Comment:
if we want to extend it to support other profilers, e.g. JFR, we'd better have an `asyncProfiler` namespace